singer-connectors/tap-twilio/requirements.txt

```diff
@@ -0,0 +1 @@
+pipelinewise-tap-twilio==1.0.0
```

Comment:
@eculicny I have selected 1.0.0 to avoid possible incapability with our old version of pipelinewise fork. I will test behaviour with latest tap-twilio version and in case it work - update a PR

Reply:
Thanks! Definitely worth being cautious. The connectors are generally well separated from the main pipelinewise application (except for the install list and config values), so updating them without updating pipelinwise usually hasn't caused us issues.

If the latest version causes problems later, we can easily roll the version back.

Reply:
The latest version works as well. Updated

Reply:
The latest version works as well. Updated